libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.java

```diff
@@ -495,4 +509,14 @@ public NavigationRoute build() {
       return new NavigationRoute(directionsBuilder.build());
     }
   }
+
+  /**
+   * Helper class to reduce redundant logging code when no other action is taken in onFailure
+   */
+  public abstract static class SimplifiedCallback implements Callback<DirectionsResponse> {
```

Comment:
I agree with @Guardiola31337 here - our developers looks at the test app for examples of how to use `NavigationRoute` and this may end up creating some confusion.